src/plugins/cld-interactive-video/index.js

```diff
@@ -0,0 +1,11 @@
+import ShoppableWidget from '../../components/shoppable-bar/shoppable-widget';
+
+export default function cldInteractivePlugin(player, options) {
+  if (options) {
+    // eslint-disable-next-line no-new
+    new ShoppableWidget(player, options);
+  }
+
+}
+
```

Comment:
@yaniv-li is this plugin being used anywhere?

Reply:
this is the shoppable plugin